best-practices/terms.md

```diff
@@ -0,0 +1,21 @@
+# Cleaning up disrespectful terms in existing systems
+
+## Checklist for renaming `master` to `main`
+
+- [ ] Create a new branch off `master`
+
+```sh
+git checkout master
+git branch -b main
+git push -u origin main
+```
+
+- [ ] On GitHub, go to **Settings** &rarr; **Branches**, and set `main` as the default branch.
```

Comment:
I often see `staging` being the default branch.
That way we avoid accidentally making PRs against `main`.

Reply:
The latest workflow we use in current projects seems to not have a staging environment, but use review apps with feature branches instead. PRs are merged directly into the default branch. @heatherabtion Is this the plan moving forward?

Reply:
@prrrnd and @runephilosof-abtion Release flows are project dependent, but CD is the preferred default. For our greenfield projects where we have short feedback loops to the end-users and can quickly resolve issues, we are using trunk based development. We are also moving clients to that flow when we see that they could benefit from the shortened feedback loop.

We would use a staging environment if:

1. A client specifically requires it for integration into their existing team's flow
2. We are pushing critical systems in a staged format where the client needs to have a QA sign-off for tagged releases (medical software, for example)
3. Systems that have greater than two week feedback cycles due to dispersed stakeholder teams or diverse integrations that require their own testing (fact would have been an example of this)

Thoughts or other considerations there?

Reply:
+1 on `main` being the default branch.